packages/frontend/src/app/app-routing.module.ts

```diff
@@ -17,13 +17,15 @@ import {Routes, RouterModule} from '@angular/router';
 import {HomeComponent} from './home/home.component';
 import {LicenseComponent} from './license/license.component';
 import {MainComponent} from './main/main.component';
+import {BuildComponent} from './build/build.component';
 
 const routes: Routes = [
   {path: '', redirectTo: '/home', pathMatch: 'full'},
   {path: 'home', component: HomeComponent},
   {path: 'license', component: LicenseComponent},
   {path: 'search', component: MainComponent},
   {path: ':org', component: MainComponent},
+  {path: ':org/:repo/:build', component: BuildComponent},
```

Comment:
I think @chingor13 brought this up today, we should probably have a prefix like:

```
/org/:org/:repo/:build
```

Otherwise, what if an org named `home` is using our product?

Reply:
Ok, switching to that. I should mention that in the marketing PR, we are dropping "/home" to use "/" though.